File: Aerolinea/src/java/airline/model/Usuario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package airline.model;

import java.util.Date;
import java.io.Serializable;

/**
 *
 * @author AndreyCh
 */
public class Usuario implements Serializable, Jsonable{
    String usuario;
    String contrasena;
    String nombre;
    String apellidos;
    String correo;
    Date fecha_nac;
    String direccion;
    String telefono;
    String celular;
    int tipo;

    public Usuario(String usuario, String contrasena, String nombre, String apellidos, String correo, Date fecha_nac, String direccion, String telefono, String celular, int tipo) {
        this.usuario = usuario;
        this.contrasena = contrasena;
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.correo = correo;
        this.fecha_nac = fecha_nac;
        this.direccion = direccion;
        this.telefono = telefono;
        this.celular = celular;
        this.tipo = tipo;
    }

    public Usuario() {
    }

    public String getUsuario() {
        return usuario;
    }

    public String getContrasena() {
        return contrasena;
    }

    public String getNombre() {
        return nombre;
    }

    public String getApellidos() {
        return apellidos;
    }

    public String getCorreo() {
        return correo;
    }

    public Date getFecha_nac() {
        return fecha_nac;
    }

    public String getDireccion() {
        return direccion;
    }

    public String getTelefono() {
        return telefono;
    }

    public String getCelular() {
        return celular;
    }

    public int getTipo() {
        return tipo;
    }

    public void setUsuario(String usuario) {
        this.usuario = usuario;
    }

    public void setContrasena(String contrasena) {
        this.contrasena = contrasena;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public void setFecha_nac(Date fecha_nac) {
        this.fecha_nac = fecha_nac;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public void setCelular(String celular) {
        this.celular = celular;
    }

    public void setTipo(int tipo) {
        this.tipo = tipo;
    }
}

File: Aerolinea/web/js/Usuario.js

function Usuario(usuario,contrasena,nombre,apellidos,correo,fecha_nac,direccion,telefono,celular,tipo){ 
  this.Usuario(usuario,contrasena,nombre,apellidos,correo,fecha_nac,direccion,telefono,celular,tipo);
}

Usuario.prototype={ 
        usuario:"",
        contrasena:"",
        nombre:"",
	apellidos:"",
	correo:"",
	fecha_nac:"",
	direccion:"",
	telefono:0,
	celular:0,
        tipo:0,
  Usuario: function(usuario,contrasena,nombre,apellidos,correo,fecha_nac,direccion,telefono,celular,tipo){
        this.usuario=usuario;
	this.contrasena=contrasena;
        this.nombre=nombre;
	this.apellidos=apellidos;
	this.correo=correo;
	this.fecha_nac=fecha_nac;
	this.direccion=direccion;
	this.telefono=telefono;
	this.celular=celular;
        this.tipo=tipo;
  },
        toString:function(){
	  return this.usuario;
	}
}
Usuario.from = function(plain){
    user = new Usuario(plain.usuario,plain.contrasena,plain.nombre,plain.apellidos,plain.correo,
                          plain.fecha_nac,plain.direccion,plain.telefono,plain.celular,plain.tipo);
    return user;
};

Usuario.to=function(user){
    return{
        _class : 'Usuario',
        usuario: user.usuario,
        contrasena: user.contrasena,
        nombre: user.nombre,
	apellidos: user.apellidos,
	correo: user.correo,
	fecha_nac: user.fecha_nac,
	direccion: user.direccion,
	telefono: user.telefono,
	celular: user.celular,
        tipo: user.tipo
    };
};